src/main/java/mate/academy/springboot/swagger/model/Product.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.swagger.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
data with entities is bad
see here why 
https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/